packages/clay-autocomplete/src/Item.tsx

```diff
@@ -34,7 +37,18 @@ const ClayAutocompleteItem: React.FunctionComponent<IProps> = ({
 	const fuzzyMatchResult = fuzzy.match(match, value, optionsFuzzy);
 
 	return (
-		<ClayDropDown.Item {...otherProps} ref={forwardRef}>
+		<ClayDropDown.Item
+			{...otherProps}
+			component={props => (
+				<a
```

Comment:
Instead of specifying a new element here, can you just do this?

```jsx
<ClayDropDown.Item
    {...otherProps}
    href="javascript:;"
>
```

Also, I'm not a huge fan of the `javascript:;` pattern. Can just use a button if its for onClick?

Reply:
Oh yes, the use case for this was that I needed the `ref` of anchor and not of `li`, but I think we can replace for a `Button` really is more sensible here.